benches/throughput.rs

```diff
@@ -20,6 +20,7 @@ use std::time::Duration;
 use criterion::{
     criterion_group, criterion_main, BenchmarkId, Criterion, SamplingMode, Throughput,
 };
+use ipnet::IpNet;
```

Comment:
The initial [failure ](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_ztunnel/559/test_ztunnel/1671242834755194880#1:build-log.txt%3A582-585)you have now is because this isn't being used.